Core/GDCore/Events/CodeGeneration/EventsCodeGenerator.cpp

```diff
@@ -506,13 +512,24 @@ void EventsCodeGenerator::CheckBehaviorParameters(
 
           if (!expectedBehaviorType.empty() &&
               actualBehaviorType != expectedBehaviorType) {
+            const auto &objectParameterMetadata =
+                instrInfos.GetParameter(lastObjectIndex);
+            // Event-functions crashes if behaviors are missing on some object
+            // of a group because they lose sight of the original objects.
+            // Missing behaviors are only "fatal" for ObjectList parameters to
+            // reduce side effects on built-in functions.
+            if (objectParameterMetadata.GetType() == "objectList") {
+              isAnyBehaviorMissing = true;
+            }

```

Comment:
I think it's safe to stop generating the instruction in this case because:
- the only built-in functions which has an `objectList` and a `behavior` are Physics collision conditions
- the diagnostic report bullied users to make them fix their groups
- the events-functions crashed so it was not usable for them